### PR TITLE
[onert/api] Enhance nnfw_train_set_traininfo description

### PR DESCRIPTION
**Pull request by zetwhite**
This PR enhances nnfw_train_set_traininfo description by adding some details.

ONE-DCO-1.0-Signed-off-by: SeungHui Youn <sseung.youn@samsung.com>

### CODE REVIEW

File: runtime/onert/api/include/nnfw_experimental.h [hunk 1]
```diff
@@ -221,13 +221,16 @@ typedef struct nnfw_train_info
...
  * @note  This function should be called after calling {@link nnfw_load_model_from_file}
  *        and before calling {@link nnfw_train_prepare}
  *
+ * @param[in] session The session to be set training infomation
+ * @param[in] info    The training infomation
+ *
  *  @return @c NNFW_STATUS_NO_ERROR If successful
  */
-NNFW_STATUS nnfw_train_set_traininfo(nnfw_session *session, const nnfw_train_info *train_info);
+NNFW_STATUS nnfw_train_set_traininfo(nnfw_session *session, const nnfw_train_info *info);
```
Comment (zetwhite) [inline runtime/onert/api/include/nnfw_experimental.h:233]: not important, just for matching *.cpp argument naming 

https://github.com/Samsung/ONE/blob/64b87a6618e3883e5ff42023de91a3cd7c05cf5d/runtime/onert/api/src/nnfw_api.cc#L391

File: runtime/onert/api/include/nnfw_experimental.h [hunk 2]
```diff
@@ -221,13 +221,16 @@ typedef struct nnfw_train_info
 } nnfw_train_info;
 
 /**
- * @brief Set training info
+ * @brief Set training infomation
  * @note  This function should be called after calling {@link nnfw_load_model_from_file}
  *        and before calling {@link nnfw_train_prepare}
  *
+ * @param[in] session The session to be set training infomation
```
Comment (glistening) [inline runtime/onert/api/include/nnfw_experimental.h:228]:
```suggestion
 * @param[in] session The session to be set training information
```

File: runtime/onert/api/include/nnfw_experimental.h [hunk 3]
```diff
@@ -221,13 +221,16 @@ typedef struct nnfw_train_info
 } nnfw_train_info;
 
 /**
- * @brief Set training info
+ * @brief Set training infomation
```
Comment (jyoungyun) [inline runtime/onert/api/include/nnfw_experimental.h:224]:
```suggestion
 * @brief Set training information
```

File: runtime/onert/api/include/nnfw_experimental.h [hunk 4]
```diff
@@ -221,13 +221,16 @@ typedef struct nnfw_train_info
...
  * @note  This function should be called after calling {@link nnfw_load_model_from_file}
  *        and before calling {@link nnfw_train_prepare}
  *
+ * @param[in] session The session to be set training infomation
+ * @param[in] info    The training infomation
```
Comment (jyoungyun) [inline runtime/onert/api/include/nnfw_experimental.h:229]:
```suggestion
 * @param[in] session The session to be set training information
 * @param[in] info    The training information
```

File: runtime/onert/api/include/nnfw_experimental.h [hunk 5]
```diff
@@ -221,13 +221,16 @@ typedef struct nnfw_train_info
...
+ *
  *  @return @c NNFW_STATUS_NO_ERROR If successful
```
Comment (glistening) [inline runtime/onert/api/include/nnfw_experimental.h:231]: Please fix it also.
```suggestion
 *  @return @c NNFW_STATUS_NO_ERROR if successful
```

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by glistening — APPROVED** [review]
LGTM

**Pull request merged**